.github/workflows/s3-deploy-legacy.yml

```diff
@@ -0,0 +1,39 @@
+name: Deploy legacy storybook to S3
```

Comment:
This `yml` file is duplicated with `s3-deploy.yml` to temporarily support the S3 deployment for v4. 

So before I merge this MR, I will turn off buddy service first.